x-pack/plugin/rank-rrf/src/yamlRestTest/resources/rest-api-spec/test/rrf/100_rank_rrf.yml

```diff
@@ -1,7 +1,7 @@
 setup:
   - requires:
-      cluster_features: "gte_v8.8.0"
-      reason: 'rank added in 8.8'
+      cluster_features: 'rrf_retriever_composition_supported'
```

Comment:
You should add a new capability. I don't think this will adequately skip stuff. Its been a while since this cluster feature was merged. 

Instead there should be a rest capability that indicates that rank, sub-searches,etc. rewritten into retrievers.

Reply:
Ok got it! I was thinking whether relying on the reworked retrievers version might be enough (especially as this affects in practice just RRF), since we'll know that all nodes can use the rewrite phase. This has only reached serverless for now, but introducing a new capability would 100% ensure consistency.

Reply:
Updated in https://github.com/elastic/elasticsearch/pull/115026/commits/9d10872f967b3164994cb337990e3fce2401fb25